docs/hardware/shockers/index.md

```diff
@@ -4,6 +4,14 @@ tags:
   - shockers
 ---
 
+!!! danger "Safety Warning"
+    **Do not wear the shocker near your neck or your heart.**  
+    Check out [Safety](../safety/safety-rules.md) for more information.
+
+!!! danger "Safety Warning" 
+    **Do not touch the pins of the shocker with both hands at the same time.** 
+    The electricity will flow through your heart. 
```

Comment:
Again here, make it a warning, and that theres a cahnce it could flow trough hear